theano/gpuarray/basic_ops.py

```diff
@@ -293,6 +293,12 @@ class GpuKernelBase(object):
     """
     params_type = gpu_context_type
 
+    def get_params(self, node):
+        # Default implementation, suitable for most sub-classes.
+        # To be necessarly overridden if a subclass uses a ParamsType
+        # (see GpuCumOp, GpuAdvancedIncSubtensor1_dev20 for examples).
+        return node.inputs[0].type.context
```

Comment:
Check if the op have a ParamsType and raise an error if so. In case the user forget to add it.